### DISCUSSION TITLE
Please don't post flag-related issues until further notice!

### DISCUSSION THREAD

**Discussion by MollyMcEvilley** [HMIS/LSASampleCode#812] [Announcements]
Hi all,

We are at a point now where the sample code will produce output that the HDX 2.0 will accept, but the HDX is generating thousands of flags for uploads that don't appear to accurately reflect the content of uploads. This is being addressed by the HDX development team.

Please don't try to validate current uploads on a flag-by-flag basis or (for now) post issues related to problems with HDX flags here -- the scale of that task goes well beyond what vendors should be asked to do and would also potentially drown out actual issues with the LSA Sample Code, which is the focus of this repository.

Please do continue to report any issues with the sample code itself if it is generating output that isn't consistent with the LSA Programming Specs. Once the broader issues with flags is addressed in the HDX, we can expand the focus of issue-tracking in GitHub to include discrepancies between the LSA specs / sample code and specific flags.

Thank you!
Molly

**Comment by cwanska** [comment]
Closing this out from 2024
